src/app.rs

```diff
@@ -53,6 +53,13 @@ pub fn build() -> App<'static, 'static> {
                 .number_of_values(1)
                 .help("Whether to use fancy or unicode icons"),
         )
+        .arg(
+            Arg::with_name("indent")
+                .short("i")
```

Comment:
This short flag is already taken by the original ls to show the inode number. I don't think it's a good idea to use the same flag for an another behavior. Can you remove it in order to keep only the long format?